shadow assets improved for a11y

chaals opened the pull request:
One SVG, only half-finished improving… (but better than the version
already merged)

Keeping in a parallel directory as protection against having it
overwritten (I hope)

Commit:
shadow assets improved for a11y

One SVG, only half-finished improving… (but better than the version
already merged)

Keeping in a parallel directory as protection against having it
overwritten (I hope)

Commit:
Merge pull request #1 from w3c/gh-pages

updating

hayatoito commented:
Instead of having a parallel directory, could you keep the updated SVG files in the same directory?
Please feel free to update SVG files directly in the directory so that we don't need to update HTML file.

We don't need to protect against having it overwritten because we'll overwrite it with a better (or latest) SVG file, always.

Commit:
image directly in place, improved

Fixed a bug that made some text invisible, put the image directly in
the assets as requested

(sorry, I got out of synch yesterday)

chaals commented:
Sorry, I will do. I somehow got out of synch yesterday, but I think this PR now does what we both want.

hayatoito commented:
Thanks. LGTM.
